Add notes on checking haystack for final beta (#7756)
desktop/desktop: docs/process/releasing-updates.md

After the release is deployed, you should monitor Desktop's [Haystack](https://haystack.githubapp.com/desktop) closely for 15 minutes to ensure no unexpected needles appear.

#### Final Beta release

> If the current version being released is the last beta before a production release, extra care should be taken when looking at Desktop's [Haystack](https://haystack.githubapp.com/desktop) roll-ups. Look out for new and unexpected errors from the past beta releases leading up to the final one. If an error is discovered bring it up with the rest of the team so we can diagnose and implement a fix.

Reviewer: Reads a little awkwardly to me, maybe? 
```suggestion
If the active beta is the last beta prior to a production release, extra care should be taken when looking at Desktop's [Haystack](https://haystack.githubapp.com/desktop) roll-ups. Look out for new and unexpected errors from the past beta releases leading up to the final one. If an error is discovered bring it up with the rest of the team so we can diagnose and implement a fix.
```
